Cross compilation of Windows failed

## Problem
Encountered issues while trying to cross-compile Windows GNU on ubuntu-latest, unable to complete the compilation.

```shell
cross build --release -p quics-client --target x86_64-pc-windows-gnu
```

### Error
```shell
The following warnings were emitted during compilation:

warning: aws-lc-sys@0.21.1: CMAKE environment variable set: cmake
warning: aws-lc-sys@0.21.1: Generating bindings - internal bindgen. Platform: x86_64-pc-windows-gnu

error: failed to run custom build command for `aws-lc-sys v0.21.1`

Caused by:
  process didn't exit successfully: `/target/release/build/aws-lc-sys-4523fb7664a5265c/build-script-main` (exit status: 101)
  --- stdout
  cargo:rerun-if-env-changed=AWS_LC_SYS_NO_PREFIX
  cargo:rerun-if-env-changed=AWS_LC_SYS_INTERNAL_BINDGEN
  cargo:rerun-if-env-changed=AWS_LC_SYS_EXTERNAL_BINDGEN
  cargo:rerun-if-env-changed=AWS_LC_SYS_NO_ASM
  cargo:rerun-if-env-changed=AWS_LC_SYS_CFLAGS
  cargo:rerun-if-env-changed=AWS_LC_SYS_PREBUILT_NASM
  cargo:rerun-if-env-changed=AWS_LC_SYS_C_STD
  cargo:rerun-if-env-changed=AWS_LC_SYS_CMAKE_BUILDER
  cargo:rerun-if-env-changed=AWS_LC_SYS_STATIC
  cargo:rerun-if-env-changed=CMAKE
  cargo:rerun-if-env-changed=CMAKE
  cargo:warning=CMAKE environment variable set: cmake
  cargo:warning=Generating bindings - internal bindgen. Platform: x86_64-pc-windows-gnu
  cargo:rerun-if-env-changed=AWS_LC_SYS_INCLUDES
  cargo:rustc-cfg=use_bindgen_generated
  cargo:rerun-if-env-changed=AWS_LC_SYS_STATIC
  cargo:rerun-if-env-changed=CMAKE_TOOLCHAIN_FILE
  cargo:rerun-if-env-changed=CMAKE_TOOLCHAIN_FILE_x86_64_pc_windows_gnu
  cargo:rerun-if-env-changed=CC
  cargo:rerun-if-env-changed=CXX
  CMAKE_TOOLCHAIN_FILE_x86_64-pc-windows-gnu = None
  CMAKE_TOOLCHAIN_FILE_x86_64_pc_windows_gnu = None
  TARGET_CMAKE_TOOLCHAIN_FILE = None
  CMAKE_TOOLCHAIN_FILE = None
  CMAKE_GENERATOR_x86_64-pc-windows-gnu = None
      make: *** [cmTC_7f655/fast] Error 2
      
      

    

    CMake will not be able to correctly generate this project.
  Call Stack (most recent call first):
    CMakeLists.txt:7 (enable_language)


  thread 'main' panicked at /cargo/registry/src/index.crates.io-6f17d22bba15001f/cmake-0.1.51/src/lib.rs:1100:5:

  command did not execute successfully, got: exit status: 1

  build script failed, must exit now
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
Error: The process 'cross' failed with exit code 101
```

--- a/.github/workflows/release-binary.yml
+++ b/.github/workflows/release-binary.yml
@@ -82,11 +82,11 @@ jobs:
             cross: true
             file-ext:
 
-          # - arch-name: i686-pc-windows-gnu
-          #   os: ubuntu-latest
-          #   target: i686-pc-windows-gnu
-          #   cross: true
-          #   file-ext: .exe
+          - arch-name: i686-pc-windows-gnu
+            os: ubuntu-latest
+            target: i686-pc-windows-gnu
+            cross: true
+            file-ext: .exe
 
           # - arch-name: i686-pc-windows-msvc
           #   os: windows-latest
@@ -94,17 +94,17 @@ jobs:
           #   cross: false
           #   file-ext: .exe
 
-          # - arch-name: x86_64-pc-windows-gnu
-          #   os: ubuntu-latest
-          #   target: x86_64-pc-windows-gnu
-          #   cross: true
-          #   file-ext: .exe
+          - arch-name: x86_64-pc-windows-gnu
+            os: ubuntu-latest
+            target: x86_64-pc-windows-gnu
+            cross: true
+            file-ext: .exe
 
-          # - arch-name: x86_64-pc-windows-msvc
-          #   os: windows-latest
-          #   target: x86_64-pc-windows-msvc
-          #   cross: false
-          #   file-ext: .exe
+          - arch-name: x86_64-pc-windows-msvc
+            os: windows-latest
+            target: x86_64-pc-windows-msvc
+            cross: false
+            file-ext: .exe
 
           - arch-name: x86_64-apple-darwin
             os: macos-latest

--- a/Cross.toml
+++ b/Cross.toml
@@ -9,7 +9,7 @@ pre-build = [
     # Use bindgen with cross
     # https://github.com/cross-rs/cross/wiki/FAQ#using-clang--bindgen
     # https://aws.github.io/aws-lc-rs/requirements/linux.html
-    "apt install --assume-yes build-essential libclang1 golang nasm",
+    "apt install --assume-yes build-essential libclang1 golang nasm wine",
     "apt install --assume-yes libclang-10-dev clang-10"
 ]
 
@@ -45,3 +45,9 @@ image = "ghcr.io/cross-rs/aarch64-unknown-linux-musl:main"
 
 [target.aarch64-linux-android]
 image = "ghcr.io/cross-rs/aarch64-linux-android:main"
+
+[target.i686-pc-windows-gnu]
+image = "ghcr.io/cross-rs/i686-pc-windows-gnu:main"
+
+[target.x86_64-pc-windows-gnu]
+image = "ghcr.io/cross-rs/x86_64-pc-windows-gnu:main"

--- a/quics-client/Cargo.toml
+++ b/quics-client/Cargo.toml
@@ -13,7 +13,7 @@ quics-protocol = { path = "../quics-protocol"}
 
 clap = { version = "4.5", features = ["std", "derive", "help", "usage"], default-features = false }
 tokio = { version = "1", features = ["rt-multi-thread", "io-util", "sync"], default-features = false }
-s2n-quic = { version = "1", features = ["provider-tls-s2n", "provider-address-token-default"], default-features = false }
+s2n-quic = { version = "1", features = ["provider-tls-default", "provider-address-token-default"], default-features = false }
 
 tracing = { version = "0.1", optional = true }
 tracing-subscriber = { version = "0.3", optional = true }

--- a/quics-server/Cargo.toml
+++ b/quics-server/Cargo.toml
@@ -11,7 +11,7 @@ quics-protocol = { path = "../quics-protocol", default-features = false }
 
 clap = { version = "4.5", features = ["std", "derive", "help", "usage"], default-features = false }
 tokio = { version = "1", features = ["rt-multi-thread", "io-util", "sync"], default-features = false }
-s2n-quic = { version = "1", features = ["provider-tls-s2n", "provider-address-token-default"], default-features = false }
+s2n-quic = { version = "1", features = ["provider-tls-default", "provider-address-token-default"], default-features = false }
 
 # https://aws.github.io/aws-lc-rs/platform_support.html
 aws-lc-rs = { version = "1", features = ["bindgen"], default-features = false, optional = true }

